framework/core/Project.pm

```diff
@@ -276,32 +277,46 @@ sub sanity_check {
     return $self->_ant_call("sanity.check");
 }
 
+
 =pod
 
-  $project->checkout_vid(vid [, work_dir])
+  $project->bugmine_sanity_check()
+
+Lightweight checking to see weather the project is correctly configured.
+
+=cut
+sub bugmine_sanity_check {
+    my $self = shift;
+    return $self->_ant_call("bugmine.sanity.check");
+}
+
+=pod
+
+  $project->checkout_vid(vid [, work_dir, is_bugmine])
 
 Checks out the provided version id (C<vid>) to F<work_dir>, and tags the the buggy AND
 the fixed program version of this bug. Format of C<vid>: C<\d+[bf]>.
 The working directory (C<work_dir>) is optional, the default is C<prog_root>.
```

Comment:
This comment needs to updated (`work_dir` should be `prog_root`, right?).

Reply:
Yes.